build fail looks real
```
======================================================================
ERROR: test_basic_app (corehq.apps.app_manager.tests.test_views.TestViews)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/app_manager/tests/test_views.py", line 140, in test_basic_app
    ], kwargs)
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/app_manager/tests/test_views.py", line 119, in _test_status_codes
    response = self.client.get(reverse(name, kwargs=kwargs), follow=False)
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/django/test/client.py", line 470, in get
    **extra)
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/django/test/client.py", line 286, in get
    return self.generic('GET', path, secure=secure, **r)
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/django/test/client.py", line 358, in generic
    return self.request(**r)
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/django/test/client.py", line 440, in request
    six.reraise(*exc_info)
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/django/views/decorators/http.py", line 41, in inner
    return func(request, *args, **kwargs)
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/domain/decorators.py", line 77, in _inner
    return view_func(req, domain_name, *args, **kwargs)
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/app_manager/views/apps.py", line 384, in view_app
    return view_generic(request, domain, app_id)
  File "submodules/dimagi-utils-src/dimagi/utils/couch/resource_conflict.py", line 24, in new_fn
    return fn(*args, **kwargs)
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/app_manager/views/view_generic.py", line 118, in view_generic
    context.update(get_app_view_context(request, app))
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/app_manager/views/apps.py", line 138, in get_app_view_context
    get_commcare_settings_layout()[app.get_doc_type()])
  File "submodules/dimagi-utils-src/dimagi/utils/decorators/memoized.py", line 11, in _inner
    return m(*args, **kwargs)
  File "submodules/dimagi-utils-src/dimagi/utils/decorators/memoized.py", line 127, in __call__
    value = self.func( *args, **kwargs)
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/app_manager/commcare_settings.py", line 74, in get_commcare_settings_layout
    for doc_type in ('Application', 'RemoteApp')
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/app_manager/commcare_settings.py", line 73, in <dictcomp>
    doc_type: _load_commcare_settings_layout(doc_type)
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/app_manager/commcare_settings.py", line 59, in _load_commcare_settings_layout
    ', '.join(settings.keys())
Exception: CommCare settings layout should mention all the available settings. The following settings went unmentioned: properties.cc-inflation-target-density
```

--- a/corehq/apps/app_manager/static/app_manager/json/commcare-settings-layout.yaml
+++ b/corehq/apps/app_manager/static/app_manager/json/commcare-settings-layout.yaml
@@ -63,6 +63,7 @@
     - hq.auto_gps_capture
     - properties.cc-login-duration-seconds
     - properties.cc-log-entity-detail-enabled
+    - properties.cc-inflation-target-density
     - hq.use_grid_menus
 
 - title: 'Advanced'

